subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/HttpElasticClient.kt

```diff
@@ -96,7 +93,16 @@ internal class HttpElasticClient(
         result
     }
 
-    private fun utcFormatter(pattern: String): DateFormat = SimpleDateFormat(pattern, Locale.getDefault()).apply {
-        timeZone = TimeZone.getTimeZone("UTC")
+    /**
+     * from javadoc:
+     *
+     * It is recommended to create separate format instances for each thread.
+     * If multiple threads access a format concurrently, it must be synchronized
+     * externally.
```

Comment:
Another good reason to use date-time API.
`java.time.format.TimeFormatter` is thread-safe.
We can use the original API in Gradle and desugar or backported version in Android code.

Reply:
yep, but we have to evaluate desugaring impact on build times